core/cat/looking_glass/cheshire_cat.py

```diff
@@ -125,29 +126,94 @@ def load_language_model(self) -> BaseLanguageModel:
         -----
         Bootstrapping is the process of loading the plugins, the natural language objects (e.g. the LLM), the memories,
         the *Main Agent*, the *Rabbit Hole* and the *White Rabbit*.
-
         """
-
+        
         selected_llm = crud.get_setting_by_name(name="llm_selected")
+        self._llm_modalities = {"image_uri": False, "image_url": False, "audio": False}
```

Comment:
`image_uri`? shouldn't it be `data_uri`?

Reply:
> `image_uri`? shouldn't it be `data_uri`?

Yes, it would be more technically correct, but I think `image_uri` is more explanatory than `data_uri`

Reply:
> Shouldn't be better to use a pydantic model rather than a dictionary, so that you avoid this kind of errors?

What kind of errors?

Reply:
> > Shouldn't be better to use a pydantic model rather than a dictionary, so that you avoid this kind of errors?
> 
> What kind of errors?

https://github.com/cheshire-cat-ai/core/pull/967/files#r1832528402